src/App.svelte

```diff
@@ -1,10 +1,12 @@
 <script>
   import { onMount } from 'svelte';
   import { getQueryParam, extractHostname, updateURL } from './utils';
+  import Header from "./Header.svelte";
   import Results from "./Results.svelte";
+  import Footer from "./Footer.svelte";
 
   export let promise;
-  const apiUrl = "https://api.checq.intercube.io";
+  const apiUrl = "https://api.checq.intercube.io/lookup";
```

Comment:
Does this endpoint work exists on the master branch?

Reply:
Good catch doesn't seem like it. It is something you added in the original pull request from task/improvements. Okay if I remove it for now?